pkg/_pub_shared/lib/search/tags.dart

```diff
@@ -73,12 +76,27 @@ abstract class PackageVersionTags {
 
   /// Package version is compatible with Dart 3.
   static const String isDart3Ready = 'is:dart3-ready';
+
+  /// Version tags that provide a positive, forward-looking property
+  /// of a prerelease or preview version.
```

Comment:
Can we extend this dartdoc with an explanation of why we only have the positive tags (eg. by explaining the consequence of having negative tags here).